website/src/pages/code-search-guide.tsx

```diff
@@ -0,0 +1,244 @@
+                        variety of developer tools.
+                    </p>
+
+                    <p>
+                        With increased complexity, we still need to efficiently write and make changes to our code while 
+                        meeting tight deadlines and stringent quality and security requirements. No pressure.
+                    </p>
+
+                    <p>
+                        On top of that, we're often dealing with knowledge silos, poor code quality, and manual processes 
+                        that interrupt flow and stall productivity. Though we'd like to pretend we're impervious to those 
+                        issues, let's be real. We're all familiar with the outcome: Development Delays (yes, with a capital
+                        D), which lead to late releases, poor quality, frustrated teams, unhappy customers, and uncompetitive
+                        products.
+                    </p>
+
+                    <p>
+                        Google and Facebook were among the first to start addressing this problem. Their solution? Invest 
+                        hundreds of millions of dollars in their customized, proprietary code search infrastructure for 
+                        internal use.
+                    </p>
+
+                    <p>
+                        But what about the rest of us?
+                    </p>
+
+                    <p>
+                        This is where universal code search (the topic of this guide) comes in.
+                    </p>
+
+                    <p>
+                        Companies like Uber, Lyft, Yelp, Qualtrics, and others adopted search technology to 
+                        enhance developer productivity in the era of Big Code. Unfortunately, a lot of devs 
+                        and companies don't know that a search engine for code even exists! We made this guide 
+                        because we love code search, and we want everyone to know why and how to get started.
+                    </p>
+
+                    <p>
+                        In the spirit of Sourcegraph's <a href="https://about.sourcegraph.com/handbook/company">
+                        open source/open product/open company ethos</a>, we're sharing advice and best practices to 
+                        help you navigate your code easier, no matter what tool(s) you may have access to.
+                    </p>
+                        
+                    <p>
+                        Our intention with this guide is to help every dev and development team navigate their codebase 
+                        with greater ease so you can spend more time creating and less time agonizing over unintended 
+                        rabbit holes. Let's get started!
+                    </p>
+
+                    <div className="container my-4 video-embed embed-responsive embed-responsive-16by9 ">
```

Comment:
@mgyorffy is this suppose to be here? When I preview the page there is a huge, blank gap between the end of the first section and the start of section 2.

Reply:
@content-hacker I think you might just be experiencing the youtube video loading slowly. That is just temporary. This is what I see.
![Screen Shot 2021-06-22 at 12 12 00 PM](https://user-images.githubusercontent.com/1383381/122985532-0297d180-d33a-11eb-81ab-80c51b24f654.png)

Reply:
Got it. These videos are not part of our guide. Can we remove them, or where did they come from?

Reply:
@content-hacker 
I think they were all in the original requirement doc. Should I remove all the videos?